target_chains/ethereum/entropy_sdk/solidity/IEntropy.sol

```diff
@@ -38,6 +38,20 @@ interface IEntropy is EntropyEvents {
         bool useBlockHash
     ) external payable returns (uint64 assignedSequenceNumber);
 
+    // As a protocol, request a random number from a provider. The protocol will also pass a secret random number
+    // in the arguments. This method will return a sequence number.
+    //
+    // When an address calls revealAndCall, see below, for the given provider and sequence number, a callback
+    // will be made to the protocol's entropyCallback method with the assigned sequence number and the random
+    // number as the arguments.
```

Comment:
I think this should say something like "The address calling this function should be a contract that inherits from the IEntropyConsumer interface. The xxx method on that interface will receive a callback with the generated random number."